.github/workflows/sql-benchmarks.yml

```
@@ -23,15 +23,18 @@ jobs:
             binary_name: tpch
             name: TPC-H on NVME
             local_dir: bench-vortex/data/tpch/1
+            targets: "datafusion:arrow,datafusion:parquet,datafusion:vortex,duckdb:parquet,duckdb:vortex"
```

Are you sure you want arrow as the initial comparison?
```suggestion
            targets: "datafusion:arrow,datafusion:parquet,datafusion:vortex,duckdb:parquet,duckdb:vortex,duckdb:duckdb"
```

---

does the order matter here?